tasks/src/main/scala/dagr/tasks/picard/DownsampleSam.scala

```diff
@@ -34,7 +34,7 @@ import scala.collection.mutable.ListBuffer
 
 object DownsamplingStrategy extends Enumeration {
   type DownsamplingStrategy = Value
-  val HighAccuracy, ConstantMemory, Chained = Value
+  val ConstantMemory, Chained, HighAccuracy = Value
```

Comment:
I guess this is ok, but I would suggest not re-ordering Enumerations in the future unless there's a really strong reason to.